src/chrome/DebuggerWebAudioDomain.ts

```diff
@@ -1,27 +1,26 @@
 import {ProtocolMapping} from 'devtools-protocol/types/protocol-mapping';
 
-export const Methods: {
-  [K in string]: keyof ProtocolMapping.Commands & `WebAudio.${K}`;
-} = {
-  disable: 'WebAudio.disable',
-  enable: 'WebAudio.enable',
-  getRealtimeData: 'WebAudio.getRealtimeData',
-};
+export enum WebAudioDebuggerMethod {
```

Comment:
When we say "Debugger", what does it exactly mean?

Reply:
Its a value passed to the [`chrome.debugger`](https://developer.chrome.com/docs/extensions/reference/debugger/#method-sendCommand) api and its defined by [chrome devtools protocol web audio domain](https://chromedevtools.github.io/devtools-protocol/tot/WebAudio/). So we could call it `WebAudioDevtoolsProtocolMethod` alternatively.

Reply:
Thanks for the explanation!